tools/debian/rules

```diff
@@ -52,6 +55,15 @@ override_dh_install:
 	# we don't need this, it contains full build paths and breaks reproducibility
 	rm -r debian/tmp/usr/lib/python*/*-packages/*.dist-info
 
+        # AppArmor profile
+ifeq ($(PRE4AA),)
+	mkdir -p debian/cockpit-ws/etc/apparmor.d/
+	install -p -m 644 tools/apparmor.d/cockpit-desktop debian/cockpit-ws/etc/apparmor.d/
+else
+	# clean up in debian unstable after broken 317-1
+	echo 'rm_conffile /etc/apparmor.d/cockpit-desktop 317-2~' > debian/cockpit-ws.maintscript
```

Comment:
Note: this is ugly and hackish, but I can't think of anything better. Directly adding that to cockpit-ws.postinst requires duplicating the condition. But that version has only been in unstable for a day, so I'll drop this again in a week after the fixed package got uploaded (I made a todo list item)